src/Illuminate/Container/Container.php

```diff
@@ -1024,6 +1026,26 @@ protected function resolvePrimitive(ReflectionParameter $parameter)
             return [];
         }
 
+        if ($this->hasConfigRepository()) {
+            foreach ($parameter->getAttributes(InjectedConfig::class) as $attribute) {
+                $attributeName = $attribute->getName();
+
+                if ($attributeName !== InjectedConfig::class) {
+                    continue;
+                }
```

Comment:
```suggestion
```

With the change in the `$parameter->getAttributes()` call, this check is no longer needed.